docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-examples/managed-guide-api.md

```diff
@@ -47,7 +47,7 @@ To create your free [sandbox](../../../cloud-basics/create-clusters/create-clust
 
 ### Determine the database version
 
-YugabyteDB Managed has [two release tracks](../../../../faq/yugabytedb-managed-faq/#what-version-of-yugabytedb-does-my-cluster-run-on), Preview for testing new features, and Stable for production deployments.
+YugabyteDB Managed has [three release tracks](../../../../faq/yugabytedb-managed-faq/#what-version-of-yugabytedb-does-my-cluster-run-on), Preview and Innovation for testing new features, and Production for production deployments.
```

Comment:
@shreyas-agnihotri wording wise -- are we okay framing Innovation track as something not suitable for production use cases?

Reply:
No, we need to update this.

@ddhodge Can you check the Release Track Descriptions in this doc (at the end) for reference?
https://docs.google.com/document/d/1uaksIU2quGOSRkFb0k-XnkNGgcoQEJObUGEElSuA6is/edit#heading=h.rwrsy7jxymxq